website/.env.sample

```diff
@@ -5,9 +5,6 @@
 # NEXT_PUBLIC_GTM_ID=GTM-123456
 
 # Toggle between local and remote graphql locations
-VERCEL_DOMAIN=localhost:3000
-#VERCEL_DOMAIN=bubs.patronage.org
-
 WORDPRESS_DOMAIN=localhost:8000
```

Comment:
Can we change the default to production? That way someone can clone site, duplicate ENV, run yarn build, and have site working. WordPress + docker should be optional for devs that actually need ACF.